src/redux-enhancers/undoRedoReducerEnhancer.ts

```diff
@@ -212,18 +212,13 @@ const undoRedoReducerEnhancer: StoreEnhancer<any> =
         return newState
       }
 
-      // if an undoable action is of type importText, update inversePatches adding latest patch of actions which contains accurate patches for importText action
+      // importText undoable action type requires a particular check before updating inverse patches as it runs a first time during app initialisation. We want to avoid this because it clutters inverse patches with a lot of unwanted actions that are specifically related with app initialisation and not particularly with NAVIGATION or other UNDOABLE actions.
+      // Cluttering inverse patches wirh unwanted actions can lead to undoing/redoing of actions such as showing up tutorial sections upon further undoing (as it is still executable beacuse of those inverse patches) which we want to avoid here.
```

Comment:
I like how you have shifted from describing what the code does, to why the code exists. Now if we can find a way to say the same thing more succinctly, it will be good to go.